batch-job.go

```diff
@@ -349,6 +423,38 @@ func (adm *AdminClient) ListBatchJobs(ctx context.Context, fl *ListBatchJobsFilt
 	return result, nil
 }
 
+// SupportedBatchJobsHeader is the header key for supported batch jobs included
+// by the server in the list batch jobs response.
+const SupportedBatchJobsHeader = "X-MinIO-Supported-Batch-Jobs"
+
+// ListBatchJobTypes lists the supported batch job types.
+func (adm *AdminClient) ListBatchJobTypes(ctx context.Context) ([]string, error) {
```

Comment:
This is what we are returning, right?

```suggestion
func (adm *AdminClient) ListBatchJobTypes(ctx context.Context) ([]BatchJobType, error) {
```